mri/optimizers/utils/cost.py

```diff
@@ -96,9 +98,12 @@ def __init__(self, gradient_op, prox_op, initial_cost=1e6,
         tolerance: float, optional
             Tolerance threshold for convergence (default is "1e-4")
         cost_interval: int, optional
-            Iteration interval to calculate cost (default is "1")
+            Iteration interval to calculate cost (default is "10")
+            if None, cost is never calculated.
         test_range: int, optional
             Number of cost values to be used in test (default is "4")
+        optimizer_type: str, default 'forward_backward'
+            specifies the type of optimizer being used
```

Comment:
can you detail the possibilities and the implications?

Reply:
Yes, Will update documentation.

And answering to your bigger question about cost PR:
Well, this PR cannot go in unless the cost issue is fixed, now we could split the PR as well.
I just added it in here as it is quite small

Reply:
I think it's a different concern and should be in a different PR.

Reply:
Sure, I can revert this PR and apply it separately, just note that we will have to merge that first

Reply:
cool